superbench/benchmarks/micro_benchmarks/cublas_function/cmd_helper.h

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Utility for storing and parsing command line arguments
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <limits>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "thirdparty/json.hpp"
```

Comment:
could you use [FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html) in cmake to embed this lib? it shouldn't be included in this project's git.

here's a [cmake example](https://github.com/nlohmann/json#embedded-fetchcontent).

Reply:
ok, thanks